src/libraries/System.Net.Quic/tests/FunctionalTests/QuicStreamTests.cs

```diff
@@ -166,11 +166,13 @@ await using (serverConnection)
             static async Task MakeStreams(QuicConnection clientConnection, QuicConnection serverConnection)
             {
                 byte[] buffer = new byte[64];
-                QuicStream clientStream = await clientConnection.OpenOutboundStreamAsync(QuicStreamType.Bidirectional);
-                ValueTask writeTask = clientStream.WriteAsync("PING"u8.ToArray(), completeWrites: true);
-                ValueTask<QuicStream> acceptTask = serverConnection.AcceptInboundStreamAsync();
-                await new Task[] { writeTask.AsTask(), acceptTask.AsTask() }.WhenAllOrAnyFailed(PassingTestTimeoutMilliseconds);
-                QuicStream serverStream = acceptTask.Result;
+
+                using QuicStream clientStream = await clientConnection.OpenOutboundStreamAsync(QuicStreamType.Bidirectional);
```

Comment:
```suggestion
                await using QuicStream clientStream = await clientConnection.OpenOutboundStreamAsync(QuicStreamType.Bidirectional);
```

I assume we're fairly inconsistent with DisposeAsync/Dispose on QuicStream in many other places, so maybe not worth changing.